ankisyncd/persistence.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import os
+from sqlite3 import dbapi2 as sqlite
+
+import anki.db
+
+class PersistenceManager:
```

Comment:
`Persistence` doesn't make it immediately clear what this class does, might be good to rename it to something clearer.

Reply:
Would `FullSyncManager` be a clearer choice?

Reply:
That's what I came up with too, might be good enough if there are no better alternatives.

Reply:
I couldn't come up with anything better, I've implemented it as that.